chsdi/views/catalog.py

```diff
@@ -87,8 +87,6 @@ def cleanNode(node):
                     path = node['path'].split('/')
                     if len(path) > 0:
                         root = path[0]
-                else:
-                    return nodes_final
```

Comment:
Are we sure about this? Why was the code there in the first place? Maybe there is a use-case that we don't see?